lib/dest/write-contents/index.js

```diff
@@ -12,6 +12,11 @@ var fo = require('../../file-operations');
 function writeContents(optResolver) {
 
   function writeFile(file, enc, callback) {
+    // Write it as a symlink
+    if (file.symlink) {
```

Comment:
I'm nervous about using the `isSymbolic()` method within vinyl-fs methods because I tried to use it in some other code I was experimenting with and it wasn't acting as I thought it would.  I found out this was due to not having a valid `stat` before I used it but I think checking this property is a better way to handle it because `isSymbolic()` can be true without having this property (I think?)